backend/internal/loki/loki.go

```diff
@@ -110,10 +110,24 @@ func GetStreamsFromResponseData(data *QueryResponseData) (Streams, error) {
 	return streams, nil
 }
 
-func GetEntriesFromStreams(streams Streams) []Entry {
-	entries := []Entry{}
+func GetEntriesFromStreams(streams Streams) []*LabeledEntry {
+	entries := []*LabeledEntry{}
 	for _, stream := range streams {
-		entries = append(entries, stream.Entries...)
+		for _, entry := range stream.Entries {
+			entries = append(entries, &LabeledEntry{Entry: entry, Labels: getFilteredLabels(stream.Labels, allowedLabels)})
+		}
 	}
 	return entries
 }
+
+var allowedLabels = []string{"ActivityType", "Name", "Schema", "Table", "Attempt", "metadata_Schema", "metadata_Table"}
```

Comment:
name, schema and table are lowercase in my worker logs

Reply:
this is loki that does this.